static/bidder-info/dianomi.yaml

```diff
@@ -0,0 +1,22 @@
+maintainer:
+  email: 'eng@dianomi.com'
+gvlVendorID: 885
+capabilities:
+  app:
+    mediaTypes:
```

Comment:
I just want to highlight that here it says you support media types `banner`, `native`, and `video` -- but in your docs PR it shows that you only support `banner` and `native`. Could you address this discrepancy?

Reply:
That's actually by design, we're _almost_ ready to support video as well, so we added video in here, and then once we are ready to support it, it will only require 1 PR to the docs repo and won't require users to update their PBS version to get video enabled. If that's a problem, then no worries, i can remove it here and deal with it later

Reply:
After some discussion, it'd be best for you to remove `video` from your .yaml file, and then just open up a new PR later when you do support it.

Reply:
We've finished adding our Video support so will updated the documentation.  This will be done in the next day or two.